docs/source/Interactive.rst

```diff
@@ -0,0 +1,43 @@
+Graphics and Interactive Modules
+*************
+
+Breakdown Diagrams
+====================
+Breakdown diagrams, such as the one shown below, help visualize things like a  vehicle’s component by component weight breakdown or an airplane’s drag breakdown.
+
+.. figure::  breakdown_example.png
```

Comment:
Commenting on the image here: would it look more like the drela diagrams if text was to the left of the lines?